mappings/net/minecraft/util/dynamic/RegistryOps.mapping

```diff
@@ -17,12 +19,28 @@ CLASS net/minecraft/class_5382 net/minecraft/util/dynamic/RegistryOps
 		COMMENT reading ops.</p>
 		COMMENT
 		COMMENT @see RegistryReadingOps#encodeOrId(Object, Object, RegistryKey, MapCodec)
+		ARG 4 allowInlineDefinitions
 	METHOD method_29761 getValueHolder (Lnet/minecraft/class_5321;)Lnet/minecraft/class_5382$class_5383;
 		ARG 1 registryRef
 	METHOD method_29763 readSupplier (Lnet/minecraft/class_5321;Lnet/minecraft/class_2385;Lcom/mojang/serialization/Codec;Lnet/minecraft/class_2960;)Lcom/mojang/serialization/DataResult;
 		COMMENT Reads a supplier for a registry element.
 		COMMENT
 		COMMENT <p>This logic is used by both {@code decodeOrId} and {@code loadToRegistry}.</p>
 		ARG 4 elementId
+	METHOD method_31150 of (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/class_5382$class_5506;Lnet/minecraft/class_5455$class_5457;)Lnet/minecraft/class_5382;
 	CLASS class_5383 ValueHolder
 		FIELD field_25513 values Ljava/util/Map;
+	CLASS class_5506 EntryLoader
+		METHOD method_31154 resourceBacked (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5382$class_5506;
+		METHOD method_31155 load (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/class_5321;Lnet/minecraft/class_5321;Lcom/mojang/serialization/Decoder;)Lcom/mojang/serialization/DataResult;
+			ARG 2 registryId
+			ARG 3 entryId
+		METHOD method_31156 getKnownEntryPaths (Lnet/minecraft/class_5321;)Ljava/util/Collection;
+			COMMENT @return A collection of file Identifiers of all known entries of the given registry.
+			COMMENT                 Note that these are file Identifiers for use in a resource manager, not the logical names of the entries.
```

Comment:
Weird indentation here

Reply:
I was trying to make it match the previous line (so it looks correct in IDE's). Should I not do that/will it be correctly formatted when applied?

Reply:
Would be better to start a new paragraph instead of trying this

Reply:
yeah, consider writing these as method doc than return tag content